index.js

```diff
@@ -129,9 +129,9 @@ client.on('message', async msg => {
 	// Non-specific commands
 	 if (msg.content.toLowerCase().startsWith("!ping")) {
 		if (online_mentor_afk_list.length < online_role.members.keyArray().length) {
-			msg.reply(`is requesting mentoring assistance ${online_role}`)
+			msg.channel.send(`<@!${msg.author.id} is requesting mentoring assistance ${online_role}`)
 		} else {
-			msg.reply("Hello, the online mentor(s) are currently away from their computer. Please be patient, they will be right back.");
+			msg.channel.send(`${msg.author}, the online mentor(s) are currently away from their computer. Please be patient, they will be right back.`);
```

Comment:
Do each of these syntaxes do the same thing? If so change to use same syntax

Reply:
Yup, those do the same. That's my bad, missed the first one